src/admin-views/settings/etp-app-banner.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Event Tickets Plus App banner section.
+ *
+ * @since 5.6.2
```

Comment:
```suggestion
 * @since TBD
 * @version TBD
```

Let's update the docs with TBD and keep note of this being moved from ETP to ET.